projectfiles/Xcode/README.md

```diff
@@ -30,33 +31,66 @@ Builds for maximum (runtime) speed and compatibility; it builds for 64-bit, with
+When compiling Wesnoth for an official release, the following steps should be taken. Packaging is separated to the 4 subchapters:
+1. In first chapter we will prepare package for all 3 distribution sources.
+2. In second chapter we will notarize Steam package and we will release it to the SteamStore.
+3. In third chapter we will notarize SourceForge package, we will create `.dmg` image and we will release it to the SourceForge.
+4. In fourth chapter we will release package to the Mac AppStore.
 
+### Packaging - Packages preparation
+ * Obtain sources from github using `git clone -b BRANCH --depth 10 https://github.com/wesnoth/wesnoth /PATH/TO/PROJECT` or use your favourite git program.
+ * Checkout to the latest tag using `cd /PATH/TO/PROJECT ; git checkout TAG` or use your favourite git program.
+ * Use `Finder` to navigate into `/PATH/TO/PROJECT/projectfiles/Xcode`.
+ * Double click on `Fix_Xcode_Dependencies` script and wait until it will be done.
+ * Compile translations as it is described in **Translations** section.
+ * Now you can open `The Battle for Wesnoth.xcodeproj` file in Xcode.
  * Update version numbers in Info.plist (if not already by the release manager).
- * Update the changelog in SDLMain.nib with `changelog.md`.
- * Rebuild translations `scons translations`.
- * Rebuild all in XCode (clean all, then build). Be sure to set it to RELEASE configuration first!
- * Download old `.dmg` release.
- * Convert it using Disk Utility to Read and Write disk image and mount it.
- * Delete old `The Battle for Wesnoth.app` and copy new `The Battle for Wesnoth.app`.
- * CodeSign `The Battle for Wesnoth.app` inside Read and Write disk image using `codesign --deep --force -s "Developer ID Application: Wesnoth, Inc (N5CYW96P9T)" The\ Battle\ for\ Wesnoth.app`. You must have Wesnoth's signing certificate.
- * Verify that you signed `.app` propertly using `spctl -a -t exec -vv The\ Battle\ for\ Wesnoth.app`.
- * Rename disk image to match new release version.
- * Unmount it and convert it back using `hdiutil convert /PATH/TO/IMAGE.dmg -format UDBZ -o /PATH/TO/NEW/IMAGE.dmg` command.
- * Sign newly created `.dmg` using `codesign -s "Developer ID Application: Wesnoth, Inc (N5CYW96P9T)" /PATH/TO/NEW/IMAGE.dmg`. You must have Wesnoth's signing certificate.
- * Verify that you signed `.dmg` propertly using `spctl -a -t open --context context:primary-signature -v /PATH/TO/NEW/IMAGE.dmg`.
- * Create SHA-256 checksum using `shasum -a 256 /PATH/TO/NEW/IMAGE.dmg` command.
- * Done! You can release it now.
+ * Update the changelog in `SDLMain.nib` with `changelog.md`.
+ * Create `dist` file using `touch /PATH/TO/PROJECT/data/dist`.
+ * You must increment build number before each distribution package. If you don't know previous build number the best way is probably procceed with steps for one of packages using build number `1` and way with which message notarization step fails. The current build number is contained here.
```

Comment:
I'd say it as "You must increment **the** build number before **creating** each distribution package. If you don't know **the** previous build number, the best way is **to proceed** with steps for one of **the** packages using build number `1` and **wait until you get an error** message **that the** notarization step **failed**. The current build number is contained **there**."
(changes bolded)